test/client/Query.test.tsx

```diff
@@ -67,10 +67,10 @@ describe('Query component', () => {
             const { client: clientResult, ...rest } = result;
 
             if (result.loading) {
-              expect(rest).toMatchSnapshot('result in render prop while loading');
+              expect(stripSymbols(rest)).toMatchSnapshot('result in render prop while loading');
```

Comment:
`stripSymbols` is not only removing all the symbols but also the checks for all the render props (see the updated snapshots in the diff). Should we remove the stripSymbols from this test to make sure that the snapshot is complete?